src/System.ServiceProcess.ServiceController/tests/System.ServiceProcess.ServiceController.TestService/TestService.cs

```diff
@@ -15,7 +15,7 @@ public class TestService : ServiceBase
         private NamedPipeServerStream _serverStream;
         private readonly Exception _exception;
 
-        public TestService(string serviceName, Exception throwException = null)
+        public TestService(string serviceName, Exception throwException = null, bool autoLog = true)
```

Comment:
I'm confused now.  This code:
https://github.com/dotnet/corefx/pull/28939/files#diff-984555e32c1d366f68dcd6312199e5e4R179
appears to be relying on autoLog defaulting to false, but it defaults to true?

Reply:
Instead of specifying it in constructor we can set this value in program.cs of the testservice app

Reply:
I removed this and now using setter to set this autolog property